### DISCUSSION TITLE
Data request: GOES-16 ABI-L2-ACMF - Advanced Baseline Imager Level 2 Clear Sky Mask Full Disk

### DISCUSSION THREAD

**Discussion by jotape-e** [microsoft/PlanetaryComputer#389] [Data request]
Hello, i would like to request/recommend, if possible, that Planetary Computer team make available the _ABI-L2-ACMF - Advanced Baseline Imager Level 2 Clear Sky Mask Full Disk_ dataset

**Comment by TaylorCorbett** [comment]
Thanks for reaching out. I am sure you have already seen that we host GOES ABI-L2 imagery in the Planetary Computer: [GOES-R Cloud & Moisture Imagery](https://planetarycomputer.microsoft.com/dataset/goes-cmi). However, ABI-L2-ACMF is not currently included as part of this collection.

Right now, we have an existing backlog of datasets to onboard, so we can't make any promises on when we could get to this. But if/when we add ABI-L2-ACMF data to this collection I'll post an update here.

**Reply by jotape-e** [reply]
Hello Mr. Corbett. Many thanks for your reply. Yes, I saw that you have a rich GOES dataset available in PC, but my need for ACMF is for project purposes. Hope to have it in the near future.

Best wishes for you and PC team and thanks for PC.
João.
